examples/light-switch-app/nrfconnect/main/AppTask.cpp

```diff
@@ -269,12 +269,15 @@ void AppTask::ButtonReleaseHandler(AppEvent * aEvent)
         case UNICAST_SWITCH_BUTTON:
             if (!sWasDimmerTriggered)
             {
-                LightSwitch::GetInstance().InitiateActionSwitch(LightSwitch::Action::Toggle);
+                LightSwitch::GetInstance().InitiateActionSwitch(LightSwitch::Action::Toggle, LightSwitch::ActionType::Unicast);
             }
             sAppTask.CancelTimer(Timer::Dimmer);
             sAppTask.CancelTimer(Timer::DimmerTrigger);
             sWasDimmerTriggered = false;
             break;
+        case GROUP_SWITCH_BUTTON:
```

Comment:
Maybe groupcast instead of group, as we already have unicast? Just to be consistent.